### PR TITLE
Kp Testing the "New Job" page.

### PR DESCRIPTION
**Pull request by kirilldko**
Testing the "New Job" page.

Notification about the absence of entered text.
Inability to click the "OK" button without entering text.

### CODE REVIEW

**Review by kirilldko — COMMENTED** [review]
refactoring, testProjectNameMissingMassage and testOkButtonIsBlocked methods

File: src/test/java/school/redrover/KPNewItemTest.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import school.redrover.runner.BaseTest;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class KPNewItemTest extends BaseTest {
```
Comment (svcat) [inline src/test/java/school/redrover/KPNewItemTest.java:11]: the name of the class should be your group name or your surname

File: src/test/java/school/redrover/KPNewItemTest.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    private final String ITEMFIELDMASSAGE = "» This field cannot be empty, please enter a valid name";
+    private final String ITEMRequiredMASSAGE = "» Required field";
```
Comment (svcat) [inline src/test/java/school/redrover/KPNewItemTest.java:13]: constants should be all in caps

File: src/test/java/school/redrover/KPNewItemTest.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import school.redrover.runner.BaseTest;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class KPNewItemTest extends BaseTest {
+    private final String ITEMFIELDMASSAGE = "» This field cannot be empty, please enter a valid name";
```
Comment (svcat) [inline src/test/java/school/redrover/KPNewItemTest.java:12]: Why do you need this "» " at the beggining of the string value?
Lines 12 & 13

File: src/test/java/school/redrover/KPNewItemTest.java [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+            String itemValidtextMassage = getDriver().findElement(
+                    By.xpath("//div[@id='itemname-required']" +
+                            "[text()='» This field cannot be empty, please enter a valid name']")).getText();
+            Assert.assertEquals(itemValidtextMassage,ITEMFIELDMASSAGE);
+        }
+    }
+
+    @Test
+    public void testOkButtonIsBlocked (){
+         boolean okButtonIsClickable = false;
```
Comment (svcat) [inline src/test/java/school/redrover/KPNewItemTest.java:46]: extra space at the beginning of the line

File: src/test/java/school/redrover/KPNewItemTest.java [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+        WebElement newItem =  getDriver().findElement(
+                By.xpath("//div[@id='tasks']//a[1]"));
+        newItem.click();
```
Comment (svcat) [inline src/test/java/school/redrover/KPNewItemTest.java:50]: I think here it's not necessary to create a variable just to click on it. It's not used anywhere else

File: src/test/java/school/redrover/KPNewItemTest.java [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+    private final By PIPELINEBYTTON = By.xpath("//span[text()='Pipeline']");
+    private final By MULCONFIGPROJBYTTON = By.xpath("//span[text()='Multi-configuration project'] ");
+    private final By SPANBYTTON = By.xpath("//span[text()='Folder'] ");
+    private final By MULTPIPELBYTTON = By.xpath("//span[text()='Multibranch Pipeline']");
+    private final By ORGANIZFOLDERBYTTON = By.xpath("//span[text()='Organization Folder']");
+
+
+    @Test
+    public void testProjectNameMissingMassage (){
+        WebElement newItem =  getDriver().findElement(
```
Comment (svcat) [inline src/test/java/school/redrover/KPNewItemTest.java:24]: I think here it's not necessary to create a variable just to click on it. It's not used anywhere else

**Review by svcat — APPROVED** [review]

**Pull request merged**